lib/Raven/ClientBuilder.php

```diff
@@ -131,6 +131,14 @@ public static function create(array $options = [])
         return new static($options);
     }
 
+    /**
+     * @return Configuration
+     */
+    public function getConfiguration()
```

Comment:
If we really want to expose the configuration from the client builder, which btw I don't find we should as all methods of that class can be called directly from the builder thanks to the `__get` magic method the method definition must be added to the interface with the appropriate description in the PHP DocBlock

Reply:
This would mean passing the client builder around instead of the simple configuration, that I think could be better. I don't want to pass around a class that has the possibility of creating a new client if it's not what I want to do.